Arrowgene.Ddon.Server/GameLogicSetting.cs

```diff
@@ -45,6 +46,28 @@ public class GameLogicSetting
         /// </summary>
         [DataMember(Order = 5)] public byte CraftConsumableProductionTimesMax { get; set; }
 
+        /// <summary>
+        /// Configures if party exp is adjusted based on level differences of members.
+        /// </summary>
+        [DataMember(Order = 6)] public bool AdjustPartyEnemyExp { get; set; }
+
+        /// <summary>
+        /// List of the inclusive ranges of (minlv, maxlv, percent). Percent is a value from 0.0 - 1.0
```

Comment:
If i understand correctly, the minlv maxlv values are level _differences_ not absolute levels right? and percent is probably more easily understood if named exp multiplier

Reply:
Yes, level differences. I can update the comments.